help.go

```diff
@@ -183,7 +110,7 @@ func ShowCommandHelpAndExit(c *Context, command string, code int) {
 func ShowCommandHelp(ctx *Context, command string) error {
 	// show the subcommand help for a command with subcommands
 	if command == "" {
-		HelpPrinter(ctx.App.Writer, SubcommandHelpTemplate, ctx.App)
```

Comment:
❓ _(blocking question)_

Can you confirm that this isn't a breaking change? I'm not sure if the CLI supports people importing this variable and changing the template that way.

Reply:
Hm no I change it back.